build.py

```diff
@@ -205,6 +205,30 @@
             pytest_args, check=True, text=True, cwd=os.path.join(pip_src, "tests")
         )
 
+        qir_test_dir = os.path.join(pip_src, "tests-qir")
+        # Try to install PyQIR and if successful, run additional tests.
+        qir_install_args = [
+            python_bin,
+            "-m",
+            "pip",
+            "install",
+            "-r",
+            "test_requirements.txt",
+        ]
+        subprocess.run(qir_install_args, check=True, text=True, cwd=qir_test_dir)
```

Comment:
Interesting.... so this has `check=True`, but if the above fails to install all the packages in `test_requirements.txt` it doesn't fail? :-/

Reply:
PyQIR has an "any" package that acts as a fallback for installation. So installation will always succeed on all platforms, but if it installed the fallback then it will fail at import time instead. @idavis was explaining this to me yesterday when I saw other Python packages that use the same fallback pattern.